pkgs/by-name/ws/wslay/package.nix

```
@@ -12,22 +20,21 @@ stdenv.mkDerivation rec {
   };
 
   strictDeps = true;
+  postPatch = ''
+    sed -i 's!add_stylesheet!add_css_file!' doc/sphinx/conf.py.in
+  '';
```

```suggestion
  postPatch = ''
    substituteInPlace doc/sphinx/conf.py.in \
      --replace-fail "add_stylesheet" "add_css_file"
  '';
```

[substituteInPlace](https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/stdenv.chapter.md#substitute-infile-outfile-subs-fun-substitute) is preferred for regex-free substitutions.

---

Thank you for your reviewing! You helped me a lot on nixpkgs PRs.